client/ayon_core/plugins/publish/extract_color_transcode.py

```diff
@@ -279,6 +280,12 @@ def process(self, instance):
             if "delete" in tags and "thumbnail" not in tags:
                 instance.data["representations"].remove(repre)
 
+            if (
+                added_review
+                and "review" not in instance.data["families"]
+            ):
+                instance.data["families"].append("review")
```

Comment:
Should not be part of this PR.

Reply:
No, it is from https://github.com/ynput/ayon-core/pull/1617